frontend/src/pages/ActivityReport/Pages/components/GoalPicker.js

```diff
@@ -85,8 +85,12 @@ const GoalPicker = ({
   const uniqueAvailableGoals = uniqBy(allAvailableGoals, 'name');
 
   // We need options with the number and also we need to add the
-  // "create new goal to the front of all the options"
+  // goal templates and "create new goal" to the front of all the options
   const options = [
+    ...availableGoals.goalTemplates.map((g) => ({
+      ...g,
+      grantIds,
+    })),
     newGoal(grantIds),
```

Comment:
What order should this be in? Create, Template, existing or template, create, existing

Reply:
I had it 
1) Template
2) Create new
3) Existing

But maybe that was taking their instructions too literally... I can follow up before we release

Reply:
Sounds like we have a decision, will make this change now